packages/riipen-ui-docs/package.json

```diff
@@ -57,6 +57,7 @@
     "@babel/core": "^7.10.5",
     "@babel/node": "^7.6.3",
     "audit-ci": "3.0.1",
+    "chokidar": "^3.4.3",
```

Comment:
I dont see this used anywhere, was it intentional?

Reply:
Ya intentional. Apparently nextjs is having issues with ubuntu in their latest patch and manually forcing this install in dev-dependencies fixes it.